cmd/ooniprobe/internal/nettests/stunreachability.go

```diff
@@ -1,13 +1,42 @@
 package nettests
 
+import (
+	"context"
+
+	engine "github.com/ooni/probe-cli/v3/internal/engine"
+	"github.com/ooni/probe-cli/v3/internal/engine/model"
+)
+
 // STUNReachability nettest implementation.
 type STUNReachability struct{}
 
+func (n STUNReachability) lookupURLs(ctl *Controller) ([]string, error) {
+	inputloader := &engine.InputLoader{
+		CheckInConfig: &model.CheckInConfig{
+			// not needed because we have default input
+		},
+		ExperimentName: "stunreachability",
+		InputPolicy:    engine.InputOrStaticDefault,
+		Session:        ctl.Session,
+		SourceFiles:    ctl.InputFiles,
+		StaticInputs:   ctl.Inputs,
+	}
+	testlist, err := inputloader.Load(context.Background())
+	if err != nil {
+		return nil, err
+	}
+	return ctl.BuildAndSetInputIdxMap(ctl.Probe.DB(), testlist)
+}
+
```

Comment:
This diff is simular to the `dnscheck` one. Also here we use the default static input if no input is provided.